matrimony_admin/views.py

```
@@ -21,10 +21,11 @@
 from U_auth.models import *
 from matrimony_admin.models import Subscription
 from U_auth.permissions import *
-from django.db.models import Count, Sum, Q
+from django.db.models import Count, Sum, F, Case, When, DecimalField
 from django.db.models.functions import TruncMonth, TruncDay
 from datetime import datetime
```

Ruff

`datetime.datetime` imported but unused

Remove unused import: `datetime.datetime`

(F401)